requirements.txt

```diff
@@ -7,6 +7,7 @@ ply==3.4
 slimit==0.8.1
 six==1.9.0
 django-appconf==1.0.1
+peewee==2.6.3
```

Comment:
This handles the database backend for the content caches. It is a light wrapper around SQLite, inspired by the Django ORM.

So far it seems pretty efficient and useable, and I've not had any of the issues that @MCGallaspy had with using the Django ORM for other content ORMing.

It also allows very easy hot swapping of database files during runtime, for, e.g., using different databases for different languages (and also channels).

That said, I'm not wedded to it, so if we can do this efficiently without adding a dependency, happy with that too.